requirements.txt

```diff
@@ -1 +1,2 @@
-redis
\ No newline at end of file
+lazy-object-proxy
```

Comment:
Can you explain what's the purpose of lazing loading the Redis connection? Is that causing any trouble for you?

Reply:
I wanted to have static Limiter object in my code, but sometimes I want to run some commands that don't need Redis connection and they should work even when Redis is not running. Without lazy, it would raise an exception.